packages/suite/src/views/wallet/trading/common/TradingSelectedOffer/TradingVerify/TradingVerifyDestinationTag.tsx

```diff
@@ -0,0 +1,49 @@
+import { ReactNode, useState } from 'react';
+
+import { Button, Column, Row, Switch } from '@trezor/components';
+import { spacings } from '@trezor/theme';
+
+import { Translation } from 'src/components/suite';
+import { useGuideOpenNode } from 'src/hooks/guide';
+import { DESTINATION_TAG_GUIDE_PATH } from 'src/views/wallet/send/Options/RippleOptions/DestinationTag';
+
+export interface TradingVerifyDestinationTagProps {
+    inputComponent: ReactNode;
+    onToggle?: (toggled: boolean) => void;
+}
+
+export const TradingVerifyDestinationTag = ({
+    inputComponent,
+    onToggle,
+}: TradingVerifyDestinationTagProps) => {
+    const { openNodeById } = useGuideOpenNode();
+
+    const [enabled, setEnabled] = useState<boolean>(false);
```

Comment:
It looks like that sometimes it can be required so take data from 

![Screenshot 2025-02-20 at 10 06 26](https://github.com/user-attachments/assets/372a37b6-6c29-4ffc-a7c4-dd970123ea5e)